tests/functional/olp-cpp-sdk-dataservice-read/VersionedLayerClientTest.cpp

```diff
@@ -118,4 +121,162 @@ TEST_F(VersionedLayerClientTest, GetPartitions) {
   EXPECT_TRUE(mock_server_client_->Verify());
 }
 
+TEST_F(VersionedLayerClientTest, GetAggregatedData) {
+  olp::client::HRN hrn(kTestHrn);
+
+  constexpr auto kTileId = "23606938";
+  constexpr auto kLayer = "testlayer";
+  constexpr auto kQuadTreeDepth = 4;
+  const auto root_tile = olp::geo::TileKey::FromHereTile(kTileId);
+  const auto tile = root_tile.ChangedLevelTo(16);
+  const auto request = olp::dataservice::read::TileRequest().WithTileKey(tile);
+
+  {
+    SCOPED_TRACE("Requested tile");
+
+    {
+      mock_server_client_->MockAuth();
+      mock_server_client_->MockTimestamp();
```

Comment:
This is not needed anymore, please check.

Reply:
Test is ok without mocking auth and timestemp requests, but it slowdown because of failing refresh auth token attempts. So I removed auth settings for the test.